<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>–ú–µ—Ä–æ–ø—Ä–∏—è—Ç–∏—è</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 245, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="list_events">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
alternate-background-color: rgb(242, 255, 235);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_but">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(221, 255, 204);
}
QPushButton:hover{
background-color: rgb(220, 255, 166);
}
</string>
    </property>
    <property name="text">
     <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å ‚Üë</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_but">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 185, 185);
}
QPushButton:hover{
background-color: rgb(255, 128, 128);
}
</string>
    </property>
    <property name="text">
     <string>–£–¥–∞–ª–∏—Ç—å üóëÔ∏è</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="event">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>760</width>
      <height>290</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>760</width>
      <height>290</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>760</width>
      <height>290</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(242, 255, 235);
}
</string>
    </property>
    <property name="text">
     <string>‚Üê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_student_but">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(221, 255, 204);
}
QPushButton:hover{
background-color: rgb(201, 255, 128);
}
</string>
    </property>
    <property name="text">
     <string>–î–æ–±–∞–≤–∏—Ç—å —Å—Ç—É–¥–µ–Ω—Ç–∞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_student_but">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 185, 185);
}
QPushButton:hover{
background-color: rgb(255, 128, 128);
}
</string>
    </property>
    <property name="text">
     <string>–£–¥–∞–ª–∏—Ç—å —Å—Ç—É–¥–µ–Ω—Ç–∞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(221, 255, 204);
}
QPushButton:hover{
background-color: rgb(201, 255, 128);
}
</string>
    </property>
    <property name="text">
     <string>–≠–∫—Å–ø–æ—Ä—Ç ‚Üì</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
